src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/LinkedEditingRange/LinkedEditingRangeEndpoint.cs

```diff
@@ -24,7 +24,10 @@ internal class LinkedEditingRangeEndpoint : IRazorRequestHandler<LinkedEditingRa
     // This is loosely based off logic from the Razor compiler:
     // https://github.com/dotnet/aspnetcore/blob/9da42b9fab4c61fe46627ac0c6877905ec845d5a/src/Razor/Microsoft.AspNetCore.Razor.Language/src/Legacy/HtmlTokenizer.cs
     // Internal for testing only.
-    internal static readonly string WordPattern = @"!?[^ <>!\/\?\[\]=""\\@" + Environment.NewLine + "]+";
+    internal static readonly string WordPattern = $"""
```

Comment:
This is so weird. I wonder if it was intended to match newlines, ie be a literal `\n`

Reply:
Yeah, it's pretty weird, and also probably incorrect to change now that I think more about it. The `Environment.NewLine` probably needs to be run on the target system.